### ISSUE TITLE
About very low resolution trainging dataset

### ISSUE THREAD

**Issue by vhank** [thstkdgus35/EDSR-PyTorch#127]
I want to train the EDSR model on my own dataset, the resolution of which is very low(e.g.50*20)，but I don't know how to set 'patch_size'.When I set a small patch_size,the loss became nan. Do you have a solution to the problem?Can I delete 'patch_size' ?
